server/routes/utils/__tests__/dataReportTestDataLarge.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
it's better to put test data(.json) in a folder

Reply:
Sure, will add to a `test_data` folder